hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/transfer/customfees/CustomRoyaltyFeeAssessor.java

```diff
@@ -138,7 +138,7 @@ private void chargeRoyalty(
                 adjustHbarFees(result, account, fee);
             } else {
                 // exchange is for token
-                adjustHtsFees(result, account, feeCollector, feeMeta, amount, denom);
+                adjustHtsFees(result, account, feeCollector, feeMeta, royalty, denom);
```

Comment:
I thought this is merged to develop 😮

Reply:
Ah it is, this is just a quirk of GitHub since I hadn't merged `develop` since branching off. 😁